### DISCUSSION TITLE
How I can implement a token?

### DISCUSSION THREAD

**Discussion by aisakk** [AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet#2452] [Q&A]
Hello, I'm new in .net and I want implement a token in .net with c#, I'm see on jwt.io/libraries this package, too see in microsoft this class JwtSecurityToken, but not explain in code how implement, you have a code to know implement in my projects?, only know that exist dotnet user-jwt but this explain to local, and i want to implement to production, i need help to implement this package in my projects, because the other packages the code is not understand.

**Comment by keegan-caruso** [comment]
duplicate of #2457
